frontend/GoodGrocer/app/components/TabBar.js

```diff
@@ -57,7 +56,7 @@ const ErrandStackScreen = () => {
       <ErrandStack.Screen
         name="ActiveErrand"
         component={ActiveErrand}
-        options={{ title: "" }}
+        options={{ title: "Active Errand" }}
```

Comment:
Does the page look bad when the title is showing? purposely left it empty because it looked strange and i believe the title is also somewhere else on the apge